lib/foreman/logging.rb

```diff
@@ -258,8 +258,18 @@ def indent_lines(string)
     end
 
     class MultilineRequestPatternLayout < MultilinePatternLayout
+      private
```

Comment:
I'm not sure `indent_lines` should also be private. Does that work?

Reply:
It works!